src/style.css

```diff
@@ -1,3 +1,83 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+img {
+  position: absolute;
+  height: 40px;
+  width: 40px;
+  margin: 10px 0 10px 50px;
```

Comment:
You can add padding: 0 50px for `header ` and align-items: center also for `header `and remove this margin